roles/dropwizard-api/tasks/gen-certs.yml

```diff
@@ -0,0 +1,39 @@
+---
+  - name: copy gen-certs.sh to dest
+    template:
+      src: gen-certs.j2
+      dest: /apis/gen-certs.sh
+    become_user: osu_apis
+
+  - name: stat gen-certs.sh
+    stat:
+      path: /apis/gen-certs.sh
+    register: gen_certs_stat
+
+  - name: run gen-certs.sh
+    command: sh /apis/gen-certs.sh
+    when: gen_certs_stat.stat.exists
+    register: script_result
+    become_user: osu_apis
+
+  - name: script debug
+    debug:
+      msg: "{{ script_result }}"
```

Comment:
@magical I guess I prefer to keep this debug message. It will be very helpful to know the current keytool files status while running this script. @floppyears what do you think about this?